app/views/MessagesView/index.tsx

```diff
@@ -48,7 +77,7 @@ class MessagesView extends React.Component {
 		this.load();
 	}
 
-	shouldComponentUpdate(nextProps, nextState) {
+	shouldComponentUpdate(nextProps: any, nextState: any) {
```

Comment:
`shouldComponentUpdate` params actually are supposed to have the same properties as the component, you can use `IMessagesViewProps` as `nextProps` type, for example.